src/java_tools/import_deps_checker/javatests/com/google/devtools/build/importdeps/DepsCheckerClassVisitorTest.java

```diff
@@ -121,6 +121,28 @@ public void testMissingMembersIgnoredWhenUnchecked() throws IOException {
     assertThat(collector.isEmpty()).isTrue();
   }
 
+  @Test
+  public void testSafelyHandleModuleInfo() throws IOException {
+    // We don't need to assert anything-- we just need to make sure the tool
+    // can safely handle odd classes like module-info without throwing any
+    // exceptions.
+    // module-info's peculiarity is that it's like java.lang.Object
+    // and doesn't have a superclass.
+    ZipFile zipFile = new ZipFile(libraryModuleInfoJar.toFile());
```

Comment:
please enclose zipFile into a try-with-resources as well, similar to classStream below